packages/gatsby-theme-docs/gatsby-node.js

```diff
@@ -33,13 +33,14 @@ exports.sourceNodes = ({ actions }) => {
       chapterTitle: String! @proxy(from: "chapter-title")
       beta: Boolean
       pagination: Boolean
-      pages: [Entry!]
+      pages: [ChapterPage!]
     }
 
-    type Entry {
+    type ChapterPage {
```

Comment:
> Not sure about the "ChapterPage" name. Wouldn't it suffice to make it a "Page" ?

It's not really a page, if any it should be `NavigationChapterPage`. It's always better to be as specific as possible when naming types.

Reply:
Right, `NavigationPage`  would be correct.  But there are no pages outside a chapter, so I think "NavigationChapterPage"  could be overspecific unless you want to futureproof the naming.

Reply:
I still think that "chapter" needs to be in the name because the `navigation.yaml` is a list of chapters (which is also how we documented it so far, both technically and visually). Therefore, it feels more appropriate to use that.

Having long names in types is not necessarily a bad thing. Also, it does not really matter at the end because we don't explicitly use the type in our code, it's more for documentation purposes and to have a more unique name within the graphql schema.

Reply:
OK, thanks for the explanation!